app/views/options.html

```diff
@@ -31,9 +31,87 @@
     </label>
     <p>Example: {{10 * options.velocityUnit().mult|longnum}} meat/{{options.velocityUnit().name}}</p>
   </div>
+<div>
+	<button id="login" style="display:none">Log in with Dropbox</button>
+	<a id="logout" href="#" style="display:none">Log out</a>
+	<button id="dbsave" style="display:none">save to Dropbox</button>
+	<button id="dbreset" style="display:none">reset Dropbox</button>
+	<script>
+		//if (window.location.href.indexOf('http://') === 0 && window.location.host.indexOf('127.0.0.1') !== 0) {
+			//window.location = window.location.href.replace('http://', 'https://');
+		//}
+
+		// Be sure to use your own app key so you can set up your own redirect URI.
+		var APP_KEY = 'k614nfbt1fp1mj7';
+		//var client = new Dropbox.Client({ key: APP_KEY });
+		var client = new Dropbox.Client({ key: APP_KEY , token: 'ajamBZAOqM4AAAAAAAABeVgXMC49WYSBWMbWGffkqCm8FJET0RApvp_6TmX6J53a' });
+
+		// First check if we're already authenticated.
+		client.authenticate({ interactive: false });
+
+		if (client.isAuthenticated()) {
+			// If we're authenticated, update the UI to reflect the logged in status.
+			loggedIn();
+		} else {
+			// Otherwise show the login button.
+			$('#login').show();
+		}
+
+function loggedIn() {
+			$('#login').hide();
+			$('#dbsave').show();
+			$('#dbreset').show();
+			var datastoreManager = new Dropbox.Datastore.DatastoreManager(client);
+
+			datastoreManager.openDefaultDatastore(function (err, datastore) {
+				if (err) { alert('opendatastore Error: ' + err); return; }
```

Comment:
Error visiting this in a new incognito window (no dropbox login, new game): https://imgur.com/AOG6Da4